src/shared-components/dropdown/index.tsx

```diff
@@ -39,10 +38,6 @@ interface DropdownProps<T> {
    */
   onDropdownContainerFocus?: (event?: React.FocusEvent) => void;
   options: T[];
-  /**
-   * Specifies maximum height of the expanded dropdown
-   */
-  optionsContainerMaxHeight?: string;
```

Comment:
@serickson-curology Now that we've removed this prop, there are [some components ](https://github.com/curology/PocketDerm/blob/79065978c1ca8899a454c32efb62ab140256cd9c/resources/assets/jsx/components/formComponents/formField/select.tsx#L55) in Pockederm that are erroring out because they are still passing it.  Is it safe to remove?